.yapfignore

```diff
@@ -0,0 +1,3 @@
+_pb2.py
+data_steward/analytics
+data_steward/lib
```

Comment:
If `lib` here refers to vendor libs then maybe we do not need this ignore as this may be GAE py2 specific.

Reply:
Hmm, dev requirements still list this:
3d. Change directory to curation/data_steward Run `pip install -r requirements.txt -t lib/`

https://docs.google.com/document/d/16Q_F5ecN1hnKviuPTbXxy914FUGFuTXO1F4zvYbjDmc/edit#

Should that be updated?

Either way, I think people probably still have this directory, so it would make sense to clean up .gitignore and .yapfignore at the same time

Reply:
The documentation is updated.  I'm fine with leaving this in because it probably does still exist in someone's personal repo.